src/opm/parser/eclipse/share/keywords/000_Eclipse100/S/SALTVD

```diff
@@ -1,4 +1,5 @@
-{"name" : "SALTVD" ,
- "sections" : ["SOLUTION"],
- "size" : {"keyword" : "EQLDIMS", "item" : "NTEQUL"},
- "items" : [{"name" : "DATA", "value_type" : "DOUBLE", "size_type" : "ALL"}]}
+{"name" : "SALTVD" , "sections" : ["SOLUTION"], "size" : {"keyword" : "EQLDIMS" , "item" : "NTEQUL"},
+        "items" : [{"name" : "DATA" ,
+                   "value_type" : "DOUBLE" ,
+                   "size_type" : "ALL",
+                   "dimension" : ["Length" , "Density"]}]}
```

Comment:
The `"dimension":` attribute will unfortunately not work; when the dimensions is given as list the lookup will be round robin to "simulat" columns - but in the case of the `SALTVD`keyword we do not know how many columns there are. Either you can ignore the dimension completely and do the transform in code, or you could try splitting as:
```
"items" : [{"name" : "DEPTH", "value_type" : "DOUBLE", "dimension" : "Length"},
               {"name" : "SALT_CONCENTRATION", "value_type" : "DOUBLE" , "dimension" : "Density", "size_type" : "ALL"]
```
hopefully that can fit in with the rest.

Reply:
This structure of this keyword is the same as RSVD. Trines suggestion for SALTVD copies the code structure of RSVD. Your comment @joakim-hove makes me a bit worried. Does this mean RSVD and RVVD is wrong as well?